mithril-aggregator/src/artifact_builder/mithril_stake_distribution.rs

```diff
@@ -7,18 +8,17 @@ use super::ArtifactBuilder;
 use crate::MultiSigner;
 use mithril_common::{
     entities::{Certificate, Epoch, MithrilStakeDistribution},
-    StdResult,
+    StdError, StdResult,
 };
 
 /// Error linked to [MithrilStakeDistributionArtifactBuilder].
 #[derive(Debug, Error)]
 pub enum MithrilStakeDistributionArtifactBuilderError {
-    /// Could not get protocol parameters
-    #[error(
-        "Could not build Mithril Stake Distribution artifact, no protocol parameters available."
-    )]
-    NoProtocolParameters(),
+    /// Protocol parameters are missing
+    #[error("Missing protocol parameter: '{0}'.")]
+    MissingProtocolParameters(StdError),
```

Comment:
Same suggestion as above:
```suggestion
    MissingProtocolParameters(Epoch),
```